lib/ansible/plugins/lookup/aws_ssm.py

```diff
@@ -24,7 +24,7 @@
         description: The region to use. You may use environment variables ar the default profile's region as an alternative.
       decrypt:
         description: A boolean to indicate whether to decrypt the parameter.
-        default: false
+        default: true
```

Comment:
https://github.com/ansible/ansible/pull/33829/files#diff-55e80dc4588cb031afcf2736c1338c47R92 has default: false

Reply:
This was an oversight on my part, I mistakenly thought that the default behavior was the same as if that was explicitly set to `false` but upon a second inspection of the code, I was incorrect. Thanks for the catch @gundalow

Reply:
line 103 (before my patch - line 106 after) has:

```
         ssm_dict['WithDecryption'] = True
```

The logic later on sets 'WithDecryption' to be False if 'decrypt' is present in terms with either valid boolean:

``
                 # decrypt the value or not
                 if key == "decrypt" and boolean(value):		                 if key == "decrypt" and boolean(value):
                     ssm_dict['WithDecryption'] = False		                     ssm_dict['WithDecryption'] = False
```

Reply:
@ajpaulson can u update the PR ?

Reply:
@daxlab update the PR with what exactly?
The original PR was to correct the documentation to match the reality of the script. 

The script always has, and still does, default WithDecryption to True and switches it to False if you pass decrypt with a valid bool.

My PR updates docstring to say the Default is True